### PR TITLE
minimal python bindings

### PR DESCRIPTION
**Pull request by svenpeter42**
I've been looking into different random forests implementation and needed python bindings for this one.

They are pretty minimal and only expose the classification forest with a sklearn-like API. Maybe they are also useful for someone else :)

### CODE REVIEW

**Comment by mnwright** [comment]
Thanks for sharing. I will try it out next week. 
Any idea how this could be tested automatically while ranger changes? I guess it's not possible to test the R and python version in one repo, is it? Would it be possible to move this to another repo (depending on the ranger repo) to include some testing with Travis? 

Another question: What is in ranger-python-extension/waf in line 168?

File: ranger-python-extension/waf [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
6$a�c�����k�Ľ]H�cj1���)�T���A&F�U���I1���s4�bi���ԌM2�f�4�3!El0��z�	Z�L�5$ȩ�-kT�Ա{�Q���wt���-��j��v(���RZ�ĪYV+cM&*j������Ɋ�i&���5�m��BU6վU�-����y�^Y2�ie�6��f٭��kf�1���iS^ut�M6���]��&�]e�m�)1+hKZ�IK��A���
:�噭{q�B��1ῌ���V��*���}_6��z���(߃���7f�),����t[��ģ�����Q��0�OR��s�r޷��Iky���kk|�h7������i��f�[)5R������v��j������V�h���<aL$<�[��__v��J�Jƶ�pd#,�P$��#,� n�D҂�>���
Ĉ�A*#3!�kKf�����ھ-7F�I-e6զٴ�ٖ�Jh�IlXdh�I�R�%���M��D�3e36�S�Q%%Ƥ��&��e2�4IMH�6$�-6F�Y6�&��*��+6�kJ#-����L�Re&b2ML�)f�6�5D��dHڒ���eSZԳI�ƔɩJ�e[%Z���wkl���i,�JJ�f��J�6�Sm��V�U����QU�j��h�Slo�Ur�IFI ,���l{ä��}b�:6�[7���7�7�7Xm�`cB��'~���烁Vb'{vг�*�K�`n{}r�:���E`�~{���2��؞o�O<8Gtz�!�
���
�F+���c][:���k(&c�]
��c�'V
c��
D2B"��Gp���-m��?*�
҆Q2�����(&JE�a�ł)Ti��C�u��s�I-/���oo����e~��	�@�M��*
�gq�uX��	��yt5�Vj�܊���F#,�>��ܓ)�!�c7a�~D�&��]@�C�%[��^�u�d-ć����Ia�`�s�q
```
Comment (mnwright) [inline ranger-python-extension/waf]: What's this?

**Comment by svenpeter42** [comment]
the thing in line 168 is the compressed self-contained build system i am using: https://github.com/waf-project/waf

**Comment by svenpeter42** [comment]
I'm pretty sure we could test both things in the same repo, but I won't have the time to implement this or maintain the bindings any further. I was just looking at different random forest implementation and probably won't be using ranger for now.

**Comment by jdonaldson** [comment]
I like ranger, and would probably use it more if it had python bindings.

**Comment by alexisdrakopoulos** [comment]
any update?

**Comment by svenpeter42** [comment]
I'm gonna close this PR because I haven't worked on this in years and the code would probably need to rewritten with all the changes since then anyway!

**Pull request closed without merging**